patches/patches.md

```diff
@@ -1,8 +1,12 @@
 This file describes the reason for the patches applied.
 
 ### react-native-vision-camera+2.15.4
-Created on **24/07/2023**
-FIXME: remove this patch after this [PR](https://github.com/mrousavy/react-native-vision-camera/pull/1666) has been merged and a new vision camera version has been released.
+Created on **24/07/2023** FIXME: remove this patch after this [PR](https://github.com/mrousavy/react-native-vision-camera/pull/1666) has been merged and a new vision camera version has been released.
+
+Updated on **01/10/2023** FIXME: remove this patch after upgrade `react-native-vision-camera` to the latest version
+
+#### Reason:
+- This patch going to remove all codes related to `colorSpace` to fix an issue on iOS 17.0.2 that cause the crash on camera load
```

Comment:
The have fixed in the v2 branch [here](https://github.com/mrousavy/react-native-vision-camera/issues/1840#issuecomment-1741734963). If they'll release `2.16.2` we could remove this patch at all. We want to add evidence of this? What do you think?